controllers/operands/kubevirt.go

```diff
@@ -270,6 +276,35 @@ func NewKubeVirt(hc *hcov1beta1.HyperConverged, opts ...string) (*kubevirtcorev1
 	return kv, nil
 }
 
+func hcoTuning2Kv(hc *hcov1beta1.HyperConverged) *kubevirtcorev1.ReloadableComponentConfiguration {
+	if hc.Spec.TuningPolicy == hcov1beta1.HyperConvergedAnnotationTuningPolicy {
+		if annotation, ok := hc.Annotations[common.TuningPolicyAnnotationName]; ok {
+
+			var rates rateLimits
+			//get values
+			err := json.Unmarshal([]byte(annotation), &rates)
+			if err != nil {
+				logger.Error(err, "failed to get qps and burst parameters")
+				return nil
```

Comment:
Since in this implementation we're dealing with the HyperConverged CR itself (not another resource like a ConfigMap), I think it would be better to exit with error here. It will also prevent setting of wrong values, as this function also called from the webhooks.

Reply:
Done. Please, take a look a it now, I'm returning an error if the `tunningPolicy` is set but the annotation is not present. I'm not sure if this is right :/